fetch.bs

```diff
@@ -1077,14 +1081,14 @@ run these steps:
  <li><p>Return <var>values</var>.
 </ol>
 
-<p>To determine if a <a>byte sequence</a> <var>value</var> is a
-<dfn>simple range header value</dfn>, perform the following steps. They return a <a>boolean</a>.
+<p>To <dfn lt="parse a single range header value|single range header value">parse a single range header value</dfn>
```

Comment:
I don't think we need to make "single range header value" able to reference this? We should also keep the ID `simple-range-header-value` working I think.

Reply:
> We should also keep the ID simple-range-header-value working I think.

:+1: I think that is what I meant to do here, but failed at the implementation 😃 Will fix on the next update.